The py-pybojc port is listed in the Portfile as dependency so it will be installed. However,
the port does not install an .egg-info file, so it appears that the package isn't present.
Removing it here from the setup.py file assures that it doesn't end up in the requires.txt
file and doesn't fail when the dependencies are checked. This is likely something that should
be fixed in the py-pyobjc port, but for now just take the pragmatic approach.
--- setup.py.orig	2021-09-04 19:02:54.000000000 -0400
+++ setup.py	2021-09-04 19:03:09.000000000 -0400
@@ -29,7 +29,6 @@
         "Operating System :: MacOS",
     ],
     install_requires=[
-        'pyobjc',
         ],
     tests_require=[
         'PyQt5',
